numpy/core/src/multiarray/mapping.c

```diff
@@ -1546,7 +1546,6 @@ _get_field_view(PyArrayObject *arr, PyObject *ind, PyArrayObject **view,
                     Py_DECREF(names);
                     return 0;
                 }
-                Py_DECREF(title);
```

Comment:
Nice catch - `title` is a borrowed reference here. But that means it should also not be `DECREF`d in the failure path just above (l.1543).

Reply:
fixed